### PR TITLE
chore: add jsx support in snap

### PR DESCRIPTION
**Pull request by khanti42**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "@commitlint/config-conventional": "^17.0.3",
     "@lavamoat/allow-scripts": "^3.0.0",
     "@lavamoat/preinstall-always-fail": "^2.0.0",
+    "@types/react": "18.0.15",
```
Comment (stanleyyconsensys) [inline package.json:28]: what is the reason to add this special version? if this is cause by type error, why other snap does not have this issue?

**Review by stanleyyconsensys — COMMENTED** [review]
overall good, but not sure why types react needed to add

File: package.json (hunk 1 above)
Comment (khanti42) [reply package.json:28]: let's remove this.

**Review by stanleyyconsensys — APPROVED** [review]
LGTM

**Pull request merged**